woocommerce/woocommerce-android
# AI product description: Celebration dialog

Fixes #9292, a subtask of #9214.

![image](https://github.com/woocommerce/woocommerce-android/assets/1522856/74f41be9-f9b6-43f8-a081-78b4da8cb371)

**To test:**
1. Go to Products -> select a product
2. Tap on Write with AI
3. Tap on Write with AI again
4. Tap on Apply
5. Notice the celebration dialog appears
6. Tap on Got it
7. Repeat steps 2-4
8. Notice the celebration dialog does not appear anymore

**Note:** Please merge #9309 first.
